portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,51 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+ // public static void main(String[] args) {
+   // ArrayList<String> messages = new ArrayList<String>();
+  //  messages.add("How are you?");
+  //  messages.add("What did you have for breakfast?");
+  //  messages.add("How old are you?");
+  //  System.out.println(messages);
+//}
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //response.setContentType("text/html;");
```

Comment:
Any reason to keep the code in comments. If you plan to clean it up later - add a TODO